doc/nrf/ug_bt_mesh_discoverable_dm_app.rst

```diff
@@ -0,0 +1,44 @@
+.. _ug_bt_mesh_discoverable_dm_app:
+
+Discovering Bluetooth mesh device in nRF Connect Device Manager
```

Comment:
Hmm, I feel this page is taken out of context. What is nRF Connect Device Manager? Why do I need my mesh device be discoverable there? It maybe clear when you start reading `Working with nRF52 Series` page, but what if you stumble upon this page when opening `Bluetooth mesh profile` page in `Libraries`? Could the whole section `FOTA over Bluetooth mesh` be moved from `ug_nrf52_developing.rst` to here?

Reply:
Funny you should say that Pavel, as I was thinking more and more in the same direction, not being happy with how it looks now. And, initially I started off with something like that, but for some strange reason ended up changing it all... there is a lot of back and forth on this one. Let's give it another go and see if we feel it sits well this time.